packages/ui/core-components/src/lib/utils.js

```diff
@@ -62,10 +62,20 @@ export const flyAndScale = (node, params = { y: -8, x: 0, start: 0.95, duration:
 
 /**
  * @param {unknown} value
- * @returns {number | null | undefined}
+ * @returns {number | undefined}
  */
 export const toNumber = (value) => {
-	if (value === null) return null;
 	if (typeof value === 'undefined') return undefined;
 	return Number(value);
 };
+
+/**
+ * @param {unknown} value
+ * @returns {boolean | undefined}
+ */
+export const toBoolean = (value) => {
+	if (typeof value === 'undefined') return undefined;
+	if (value === 'true') return true;
+	if (value === 'false') return false;
```

Comment:
Should this be case-insensitive

Reply:
[4dc833b](https://github.com/evidence-dev/evidence/pull/2184/commits/4dc833b7648890167e6b79f52caa0767d55aa3c4)